CRADLE/CorrectBiasStored/calculateOneBP.pyx

```diff
@@ -66,130 +64,95 @@ cpdef performRegression(trainingSet, scatterplotSamples, covariates, ctrlBWNames
 			currentRow += (trainingRegion.analysisEnd - trainingRegion.analysisStart)
 	#### END Get X matrix
 
-	#### Get Y matrix
-	cdef double [:] yView = np.zeros(trainingSet.xRowCount, dtype=np.float64)
+	#### Initialize COEF arrays
+	COEFCTRL = np.zeros((len(ctrlBWNames), COEF_LEN), dtype=np.float64)
+	COEFEXPR = np.zeros((len(experiBWNames), COEF_LEN), dtype=np.float64)
+
+	for i, bwFileName in enumerate(ctrlBWNames):
+		readCounts = getReadCounts(bwFileName, trainingSet, ctrlScaler[i])
+		model = buildModel(readCounts, xView)
```

Comment:
Would this significantly increase memory requirement by copying xView? xView is a huge matrix with ~10^6 rows and I am concerned if this may require more memory in running the script.

Reply:
10^6 64-bit values isn't all that big -- it's only about 8 megabytes. 
```python
>>> a = np.zeros(1_000_000, dtype=np.float64)
>>> a.nbytes
8000000
```

Reply:
but also, the value isn't being copied. a reference to the value is being returned.